examples/gltf-animation.rs

```diff
@@ -0,0 +1,39 @@
+extern crate three;
+
+fn main() {
+    let shaders = concat!(env!("CARGO_MANIFEST_DIR"), "/data/shaders");
+    let mut win = three::Window::builder("Three-rs glTF example", &shaders).build();
+    let mut light = win.factory.directional_light(0xFFFFFF, 7.0);
+    light.look_at([1.0, 1.0, 1.0], [0.0, 0.0, 0.0], None);
+    win.scene.add(&light);
+    win.scene.background = three::Background::Color(0xC6F0FF);
+
+    let default = concat!(env!("CARGO_MANIFEST_DIR"), "/test_data/Lantern.gltf");
+    let path = std::env::args().nth(1).unwrap_or(default.into());
+    let (group, mut cameras, _meshes, clips) = win.factory.load_gltf(&path);
```

Comment:
I'm voting for new struct. Tuples are cool, but not in such situations.

Reply:
I'm inclined to agree. What does @kvark think?